h2o-core/src/main/java/water/util/MRUtils.java

```diff
@@ -130,6 +130,175 @@ public void map(Chunk[] cs, NewChunk[] ncs) {
     @Override public void reduce( ClassDist that ) { ArrayUtils.add(_ys,that._ys); }
   }
 
+  /**
+   * compute the number of rows in a frame.
+   */
+  public static class CountFrameRows extends MRTask<CountFrameRows> {
```

Comment:
@wendycwong, you don't need to use an MR task, the same information is available in Vec's ESPC.